dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -447,6 +435,21 @@ error.missing.required.attribute.singleton=CWWKG0095E: The element {0} is missin
 error.missing.required.attribute.singleton.explanation=The missing attribute is required in the configuration file server.xml or one of its referenced configuration files.
 error.missing.required.attribute.singleton.useraction=Add the attribute to the configuration file server.xml or one of its referenced configuration files.
 
+#"ibm:extendsAlias" is an xml attribute name and should not be translated
+error.factoryOnly.extendsAlias=CWWKG0096E: The {0} persisted identity is not a factory persisted identity and so cannot have an ibm:extendsAlias attribute of {1}. Reconfigure the persisted identity to use a factory persisted identity or remove the ibm:extendsAlias attribute from the persisted identity.
```

Comment:
**change**
 The {0} persisted identity is not a factory persisted identity and so cannot have an ibm:extendsAlias attribute of {1}
**to**
 The {0} persisted identity cannot have an ibm:extendsAlias attribute of {1} because it is not a factory persisted identity.

Reply:
THese messages didn't actually change -- they were listed in the wrong part of the file so I moved them. Given our guidelines against changing messages in service without a compelling reason, they should probably stay the way they are.